src/main/kotlin/com/amazon/elasticsearch/replication/task/shard/TranslogSequencer.kt

```diff
@@ -70,7 +71,16 @@ class TranslogSequencer(scope: CoroutineScope, private val replicationMetadata:
                 replayRequest.parentTask = parentTaskId
                 launch {
                     var relativeStartNanos  = System.nanoTime()
-                    val replayResponse = client.suspendExecuteWithRetries(replicationMetadata, ReplayChangesAction.INSTANCE, replayRequest, log = log)
+                    val retryOnExceptions = ArrayList<Class<*>>()
+                    retryOnExceptions.add(MappingNotAvailableException::class.java)
```

Comment:
Nit - can we use Collections.singletonList()

Reply:
the function is expecting ArrayList. There will be type mismatch with Collections.singletonList()